README.md

```diff
@@ -178,14 +187,15 @@ Nabla is built with interoperation in mind, supporting memory export and import
 [TODO]:
 - Cancellable Future based Async I/O
 - Virtual File System (archive mounting, our alternative to #embed, everything is referenced by absolute - path)
-
+- IUtiltities  Using Fixed-sized staging memory for easier cpu-gpu transfers? format promotion?
```

Comment:
link your blog

Reply:
you commented on outdated commit it seems, already linked the blog a day ago, It's under `Data Transfer Utilities`